src/modules/mealDisplay.js

```diff
@@ -0,0 +1,103 @@
+import popupComment from './display-popup-comment';
+import count from './items-count';
+import newLike from './new-like';
+
+async function fetchLikesData(apikey, likeid) {
+  try {
+    apikey = 'tnE2k6P5BdZ2HCTjbd0V';
+    const response = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${apikey}/likes?`);
+    const data = await response.json();
+    return data.find((element) => element.item_id === likeid);
+  } catch (error) {
+    console.error('Error fetching likes data:', error);
+    return null;
+  }
+}
+
+function createLikeCountElement(likeData) {
+  const likeCount = document.createElement('span');
+  likeCount.innerHTML = `likes:${likeData.likes}`;
+  return likeCount;
+}
+
+function handleLikeClick(like, likeData, likeCount, likeid) {
+  like.addEventListener('click', async () => {
+    if (likeData) {
+      likeData.likes += 1;
+      likeCount.innerHTML = `likes:${likeData.likes}`;
+    }
+    await newLike(likeid);
+  });
+}
+
+function createMealElement(mealData, popupCallback) {
```

Comment:
As per the project requirements, you should be using ES6 features, that include the use of arrow functions instead of the old function declaration way. Kindly update your code accordingly.